paddle/fluid/framework/ir/pass.cc

```diff
@@ -40,6 +41,10 @@ Graph* Pass::Apply(Graph* graph) const {
   PADDLE_ENFORCE(VarDescIsConsistency(*graph),
                  "The VarDescs of persistable variable are not consistency.");
   applied_ = true;
+  if (!graph->Has(kPassRecorder)) {
```

Comment:
No one use it also?

Reply:
Maybe, I will fix this in the next [PR](https://github.com/PaddlePaddle/Paddle/pull/18833).